routes/console.php

```diff
@@ -25,15 +25,16 @@
         $user->name = $this->ask('Name (with surname)');
     } while (!$user->name);
     do {
-        $user->email = $this->ask('E-mail');
-        if (User::query()->where('email', $user->email)->exists()) {
+        if ($user->email) {
             $this->error('User already exists');
-            $user->email = null;
         }
-    } while (!$user->email);
-    $user->email_verified_at = $this->confirm('Mark e-mail as confirmed', true) ? (new DateTimeImmutable()) : null;
-    $password = $this->secret('Password');
-    $user->password = $password ? Hash::make($password) : null;
+        $user->email = $this->ask('E-mail (optional)');
+    } while ($user->email && User::query()->where('email', $user->email)->exists());
+    if ($user->email) {
+        $user->email_verified_at = $this->confirm('Mark e-mail as confirmed', true) ? (new DateTimeImmutable()) : null;
+        $password = $this->secret('Password');
+        $user->password = $password ? Hash::make($password) : null;
+    }
```

Comment:
w sumie nie lepiej by napisać te commandy w osobnych klasach? https://laravel.com/docs/10.x/artisan#command-structure

Reply:
to jest i tak docelowo do usunięcia